docs/en/administration/fault_diagnosis_and_analysis.md

```diff
@@ -60,20 +60,19 @@ cat /var/log/syslog | grep 'juicefs' | grep '<ERROR>'
 
 ### Kubernetes CSI Driver
 
-Depending on the version of the JuiceFS CSI Driver you are using, there will be different ways to obtain logs. For details, please refer to [CSI Driver documentation](https://juicefs.com/docs/csi/troubleshooting).
+Depending on the version of the JuiceFS CSI Driver you are using, there will be different ways to retrieve logs. For details, please refer to [CSI Driver documentation](https://juicefs.com/docs/csi/troubleshooting).
```

Comment:
```suggestion
Depending on the version of the JuiceFS CSI Driver, there are different ways to retrieve logs. Please refer to [CSI Driver documentation](https://juicefs.com/docs/csi/troubleshooting) for details.
```
- It is a statement, so I think we can use the present tense here.

Reply:
can we ditch the "please" in all future work? not just to make things shorter, tech docs (that I've seen) always use neutral tone.